public/index.html

```diff
@@ -7,6 +7,7 @@
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Web site created using create-react-app" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <link href="https://fonts.googleapis.com/css?family=Lato|Dosis" rel="stylesheet">
```

Comment:
This is good. However, if you want to get the best performance, you could read this [article](https://blog.larsbehrenberg.com/the-best-way-to-load-and-use-google-fonts-with-react-gatsby-and-nextjs) that explains a better approach to avoid fonts loading being a render-blocking issue.